pyproject.toml

```diff
@@ -0,0 +1,128 @@
+# adpated from https://github.com/pola-rs/polars/blob/main/py-polars/pyproject.toml
```

Comment:
I don't think we should change tooling in an update PR.

Reply:
Thanks for taking a look, but I do not understand what you mean here. Should this be removed altogether?

Reply:
I am assuming that you are suggesting removal, which is now done.